packages/components/src/range-control/README.md

```diff
@@ -1,17 +1,101 @@
 # RangeControl
 
-RangeControl component is used to create range slider to input numerical values.
+RangeControls are used to make selections from a range of incremental values.
 
+![](https://make.wordpress.org/design/files/2018/12/rangecontrol.png)
 
-## Usage
+A RangeControl for volume
+
+## Table of contents
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+## Design guidelines
+
+### Anatomy
+
+![](https://make.wordpress.org/design/files/2018/12/rangecontrol-anatomy.png)
+
+A RangeControls can contain the following elements:
+
+1. **Track**: The track shows the range available for user selection. For left-to-right (LTR) languages, the smallest value appears on the far left, and the largest value on the far right. For right-to-left (RTL) languages this orientation is reversed, with the smallest value on the far right and the largest value on the far left.
+2. **Thumb**: The thumb slides along the track, displaying the selected value through its position.
+3. **Value entry field** (optional): The value entry field displays the currently selected, specific numerical value.
+4. **Icon** (optional): An icon can be displayed before or after the range slider.
+5. **Tick mark** (optional): Tick marks represent predetermined values to which the user can move the slider.
+
+### Behavior
+
+- **Click and drag**: The range slider is controlled by clicking the thumb and dragging it.
+- **Click jump**: The range slider is controlled by clicking the track.
+- **Click and arrow**: The continuous slider is controlled by clicking the thumb, then using arrow keys to move it.
+- **Tab and arrow**: The range slider is controlled by using the tab key to select the thumb of the desired slider, then using arrow keys to move it.
+- **Tick marks** (Optional) Discrete sliders can use evenly spaced tick marks along the slider track, and the thumb will snap to them. Each tick mark should change the setting in increments that are discernible to the user.
+- **Value entry field** (Optional): Discrete sliders have value entry fields. After a text entry is made, the slider position automatically updates to reflect the new value.
+
+### Usage
+
+RangeControls reflect a range of values along a track, from which users may select a single value. They are ideal for adjusting settings such as volume, opacity, or text size.
```

Comment:
This is a broader question but I wonder whether it makes more sense to refer to `RangeControls reflect` or the singular `A RangeControl reflects`. Might be worth a copy review on this.